erpc.yaml

```diff
@@ -26,377 +34,126 @@ projects:
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 42161
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 137
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 8453
-        failsafe:
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 10
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1284
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
-      - architecture: evm
-        evm:
-          chainId: 1285
-        failsafe:
-          circuitBreaker: ~
-          timeout:
-            duration: 60s
-          retry:
-            maxCount: 3
-            delay: 500ms
-            backoffMaxDelay: 10s
-            backoffFactor: 0.3
-            jitter: 200ms
-          hedge:
-            delay: 1000ms
-            maxCount: 2
+      allowedMethods:
+        - 'GET'
+        - 'POST'
+        - 'OPTIONS'
+
+    networkDefaults:
+      failsafe:
+        circuitBreaker: ~
+        timeout:
+          duration: 60s
+        retry:
+          maxAttempts: 3
+          delay: 500ms
+          backoffMaxDelay: 10s
+          backoffFactor: 0.3
+          jitter: 200ms
+        hedge:
+          delay: 1000ms
+          maxCount: 2
+
+    upstreamDefaults:
+      jsonRpc:
+        supportsBatch: true
+        batchMaxWait: 100ms
+        batchMaxSize: 100
```

Comment:
@lyoungblood  If there's no explicit reason to use batching I recommend to remove this because generally batching doesn't bring much benefit in terms of costs, but it creates loads of coordination/delay/error handling issues. https://docs.erpc.cloud/operation/batch

Reply:
Sounds good, I'll remove this in a subsequent PR